pkg/actors/actors.go

```diff
@@ -904,7 +918,7 @@ func (m *ActorMetadata) removeReminderFromPartition(reminderRefs []actorReminder
 }
 
 func (m *ActorMetadata) insertReminderInPartition(reminderRefs []actorReminderReference, reminder *Reminder) ([]Reminder, actorReminderReference, string, *string) {
-	newReminderRef := m.createReminderReference(reminder)
+	newReminderRef, _ := m.createReminderReference(reminder)
```

Comment:
Could we skip this error safely?

Reply:
> Could we skip this error safely?

I think this cannot be ignored, it has been modified.